src/themes/default/store/inspirations.js

```diff
@@ -0,0 +1,63 @@
+// initial state
```

Comment:
1. Is it the same data format as the one provided by Instagram API? If not please change it so it can mock the Instagram API more accurate.
2. Data stores not related to UI should be in `src/store` folder. 
You can pull out the data itself from vuex store and put it in `resource` folder as json file.

Reply:
@filrak  

1. This and naming the block with anything related to Instagram would be limiting, I think we can fetch images from different social media, piping them through an aggregator of sorts and then saving them in the common store. In this way we could share images from Twitter, IG, FB and more. What do you think?  

2. I'll get to it ASAP

Reply:
1. What i ment was just mocking it properly. The json from `resource` folder should look like data retrieved from Instagram API. Of course you can use this json in store as you wrote (new one).